gammapy/estimators/parameter.py

```diff
@@ -42,10 +42,10 @@ class ParameterEstimator(Estimator):
             * "ul": estimate upper limits.
             * "scan": estimate fit statistic profiles.
 
-        Default is None so the optional steps are not executed.
-    fit : `~gammapy.modeling.Fit`
-        Fit instance specifying the backend and fit options.
-    reoptimize : bool
+        Default is None, so the optional steps are not executed.
+    fit : `~gammapy.modeling.Fit`, optional
+        Fit instance specifying the backend and fit options. Default is None, so "minuit" is used.
```

Comment:
Well, technically default internally instantiate `Fit` without option here. 
"Fit instance specifying the backend and fit options. If None, the `Fit` instance is created internally. Default is None."